hmpps-clamav/Dockerfile.freshclammed

```diff
@@ -1,9 +1,13 @@
 FROM ghcr.io/ministryofjustice/hmpps-clamav:latest
 
-USER "root"
+USER root
+RUN chown -R clamav:clamav /var/lib/clamav
+
+USER clamav
 RUN freshclam
 
 # We run as UID 1000 in kubernetes, this will allow us to start the process
+USER root
 RUN mkdir -p /var/run/clamav && \
```

Comment:
You actually dont need anything in this dockerfile except RUN freshclam.
Running freshclam as root overwrote the correct ownership of the dir

Reply:
Oh cool, if that's the case, let's go with that! 😄